tests/RedisConnectionTest.php

```
@@ -158,4 +158,115 @@ public function testDeleteHandlesConnectionFailure(): void
+        // キーの数が期待通りであることを確認
+        // 注: 他のテストで作成されたキーも含まれる可能性があるため、
+        // 作成したキーが含まれていることのみを確認
+        foreach ($testKeys as $key) {
+            self::assertContains($key, $keys);
+        }
+
+        // クリーンアップ
+        foreach ($testKeys as $key) {
+            $connection->delete($key);
+        }
+    }
```

PHPMD (2.15.0)

The method testKeysReturnsDeduplicated() has a Cyclomatic Complexity of 10. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)